### DISCUSSION TITLE
Scope is hard to maintain

### DISCUSSION THREAD

**Discussion by ghost** [NovaoTech/solaris#47] [General]
This project is pretty much impossible to do fully, it requires too many resources and to finish it, it's not super easy.

I'm not entirely sure if this should aim to have a community, but only the project sharing. Thoughts?

**Comment by icarusinthewings** [comment]
Project sharing is #1 priority, community comes after that.

**Reply by ghost** [reply]
That doesn't change the fact that the full scope of this is extremely hard to achieve.
